sdk/python/packages/flet-core/src/flet_core/geolocator.py

```diff
@@ -245,3 +256,56 @@ async def open_location_settings_async(
             wait_timeout=wait_timeout,
         )
         return opened == "true"
+
+    def service_enable(self, wait_timeout: Optional[float] = 10) -> bool:
+        opened = self.invoke_method(
+            "service_enable",
+            wait_for_result=True,
+            wait_timeout=wait_timeout,
+        )
+        return opened == "true"
+
+    async def service_enable_async(
+        self, wait_timeout: Optional[float] = 10
+    ) -> bool:
+        opened = await self.invoke_method_async(
+            "service_enable",
+            wait_for_result=True,
+            wait_timeout=wait_timeout,
+        )
+        return opened == "true"
+
+    def service_disable(self, wait_timeout: Optional[float] = 10) -> bool:
+        opened = self.invoke_method(
+            "service_disable",
+            wait_for_result=True,
+            wait_timeout=wait_timeout,
+        )
+        return opened == "true"
+
+    async def service_disable_async(
+        self, wait_timeout: Optional[float] = 10
+    ) -> bool:
+        opened = await self.invoke_method_async(
+            "service_disable",
+            wait_for_result=True,
+            wait_timeout=wait_timeout,
+        )
+        return opened == "true"
+
+    @property
+    def on_position(self) -> OptionalControlEventCallable:
+        return self._get_event_handler("position")
+
+    @on_position.setter
+    def on_position(self, handler: OptionalEventCallable["PositionEvent"]):
+        self.__on_position.subscribe(handler)
+        self._set_attr("onPosition", True if handler is not None else None)
+
```

Comment:
We've just merged #3808 changing the way strongly-typed events defined/work.

Could you please update `on_position` event getter/setter like here: https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet-core/src/flet_core/dismissible.py#L248-L256

Reply:
Done